CHANGELOG.adoc

```diff
@@ -29,6 +29,7 @@ For a detailed view of what has changed, refer to the {uri-repo}/commits/master[
 * Added `--disable-active-clipboard` switch to prevent clipboard request injection
 * Added `--no-downgrade` switch to prevent protocol downgrading where possible {uri-issue}189[#189]
 * Added `--no-files` switch to prevent extracting transferred files {uri-issue}195[#195]
+* Added `--gdi` MITM switch to enable accelerated graphics pipeline (MS-RDPEGDI) ({uri-issue}50[#50])
```

Comment:
I do think GDI should be supported by default, but I did want to gate it at first since I'm sure there are unidentified bugs. If you believe we should go straight into GDI (and fix bugs as they come), then sure, I can flip it around.

Reply:
How about this: we merge this as-is, we open a ticket to consider flipping that behavior and we milestone that ticket with `vNext`.

So we will benefit from this code `$now`, we deploy master in our usual spot :wink: and our users can do to. Before our next release, we evaluate whether this is ready enough to be flipped or not.

What do you think?

Reply:
I'd argue that since the MITM does not parse GDI, there is very little chance that this PR breaks the MITM and that any bug can be fixed with minor version releases, so it should be the default

Reply:
I think @obilodeau has the right approach. We're still a while off from releasing, so we can land this and revisit making GDI default when we're ready for 1.0.

I also agree that any bugs that come up are going to be purely aesthetic/rendering bugs, since recording is unaffacted. The problem is I don't want to become a blocker for people with video that  needs to be viewed.

Reply:
Good. Opened #209.